src/deploy/NVA_build/noobaa_init.sh

```diff
@@ -178,7 +178,7 @@ init_endpoint() {
   fix_non_root_user
 
   cd /root/node_modules/noobaa-core/
-  run_internal_process node ./src/s3/s3rver_starter.js
+  run_internal_process node ${ENDPOINT_NODE_OPTIONS} ./src/s3/s3rver_starter.js
```

Comment:
Please make a comment or on how can it be used

Reply:
Done. Added to the PR comment

Reply:
Added comment to the file as well

Reply:
Greate, Just 2 notes: 
1. change the `k edit...` to `kubectl edit...`
2. We can have a CLI flag for that, WDYT?

Reply:
1. Done
2. We can do that, but since it's not a small diff, in my opinion, it should be additional task and PR.

Reply:
> 2. We can do that, but since it's not a small diff, in my opinion, it should be additional task and PR.

of course, as it is on the operator side...